app/assets/javascripts/automate_import_export.js

```diff
@@ -118,10 +118,10 @@ var Automate = {
 
   setUpAutomateImportClickHandlers: function() {
     var tearDownGitImportOptions = function() {
-      $('.git-branches').find('option').remove().end();
-      $('.git-tags').find('option').remove().end();
-      $('.git-branches').selectpicker('refresh');
-      $('.git-tags').selectpicker('refresh');
+      $('select.git-branches').find('option').remove().end();
```

Comment:
@eclarizio - Can we include in the description of the PR why this change was needed?